scrapers/MindGeekAPI.py

```diff
@@ -134,9 +126,9 @@ def graphql_findTagbyName(name):
 def graphql_createMarker(scene_id, title, main_tag, seconds, tags=[]):
     main_tag_id = graphql_findTagbyName(main_tag)
     if main_tag_id is None:
-        debug("The 'Primary Tag' don't exist ({}), marker won't be created.".format(main_tag))
+        log.error("The 'Primary Tag' don't exist ({}), marker won't be created.".format(main_tag))
         return None
-    debug("[DEBUG] Creating Marker: {}".format(title))
+    log.debug("Creating Marker: {}".format(title))
```

Comment:
```suggestion
    log.info("Creating Marker: {}".format(title))
```
It's a useful information